spring-cloud-skipper-platform-cloudfoundry/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-cloud-skipper-platform-cloudfoundry</artifactId>
+    <packaging>jar</packaging>
+    <name>Spring Cloud Skipper :: CloudFoundry Platform</name>
+    <parent>
+        <artifactId>spring-cloud-skipper-parent</artifactId>
+        <groupId>org.springframework.cloud</groupId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <spring-cloud-deployer-cloudfoundry.version>1.3.0.M6</spring-cloud-deployer-cloudfoundry.version>
+        <reactor.version>3.0.7.RELEASE</reactor.version>
```

Comment:
This now revealed that our build structure is not optimal. If package rely on `spring-cloud-skipper-parent`, versions should come from there. Not sure if this should be fixed in this PR but eventually we need to go through all poms.